app/components/shared/SearchablePages.tsx

```diff
@@ -25,9 +25,9 @@ class SearchablePagesProps {
  */
 @Component({ props: createProps(SearchablePagesProps) })
 export default class SearchablePages extends TsxComponent<SearchablePagesProps> {
-  currentPage: string;
+  currentPage: string = '';
   pagesInfo: Dictionary<IPageInfo> = null;
-  searchResultPages: string[];
+  searchResultPages: string[] = null;
```

Comment:
Should this be empty array instead?  Generally empty array makes more sense and avoids accidental null errors.